electron_main.js

```diff
@@ -1,8 +1,19 @@
-var BrowserWindow = require('electron').BrowserWindow
-var app = require('electron').app
+// compatible with both Electron 0.x and 1.x
+try {
+  var BrowserWindow = require("browser-window");
+} catch {
```

Comment:
@RSATom nothing, I'm just not used to coding vanillaJS and I don't know electron that well.  I'll update this to be more in line with your style.